### ISSUE TITLE
Dimmers

### ISSUE THREAD

**Issue by 4tetra4** [soler2000/com.lightwaverf#16]
Hi Andy,

I have just installed a couple of dimmers. I can add them as devices and can turn them on but can`t set dim level or turning them off.

**Comment by soler2000** [comment]
Please try again with the latest version,  I have just tested with mine and its working

**Comment by 4tetra4** [comment]
Still can`t turn off the the lights connected to the dimmer. App:0.4 Firmware:0.8.32

**Comment by soler2000** [comment]
Can you try deleting the old device and readding them

**Comment by 4tetra4** [comment]
I did that it did not work. Now I have reseted and wiped homey, reinstalled apps and added devices. The issue is still the same: can turn it on, can dim it but can`t turn it off. I have a JSJSLW420SS dimmer and and the app has been installed from the add devices menu. Turning off dimmers does not work on the pairing testing page, in the devices menu nor in the flows. The dimmer is new but it works even with the old Siemens remotes.
